migrations/20180903142820_renaming_fk_columns_in_db.js

```diff
@@ -0,0 +1,154 @@
+exports.up = function(knex) {
+  let promiseArray = [];
+
+  promiseArray.push(
+    knex.schema.table("Answers", t => {
+      t.renameColumn("QuestionPageId", "questionPageId");
+    })
+  );
+
+  promiseArray.push(
+    knex.schema.table("Options", t => {
+      t.renameColumn("AnswerId", "answerId");
+    })
+  );
+
+  promiseArray.push(
+    knex.schema.table("Pages", t => {
+      t.renameColumn("SectionId", "sectionId");
+    })
+  );
+
+  promiseArray.push(
+    knex.schema.table("Routing_ConditionValues", t => {
+      t.renameColumn("OptionId", "optionId");
+      t.renameColumn("ConditionId", "conditionId");
+    })
+  );
+
+  promiseArray.push(
+    knex.schema.table("Routing_Conditions", t => {
+      t.renameColumn("RoutingRuleId", "routingRuleId");
+      t.renameColumn("QuestionPageId", "questionPageId");
+      t.renameColumn("AnswerId", "answerId");
+    })
+  );
+
+  promiseArray.push(
+    knex.schema.table("Routing_Destinations", t => {
+      t.renameColumn("PageId", "pageId");
+      t.renameColumn("SectionId", "sectionId");
+    })
+  );
+
+  promiseArray.push(
+    knex.schema.table("Routing_RuleSets", t => {
+      t.renameColumn("QuestionPageId", "questionPageId");
+      t.renameColumn("RoutingDestinationId", "routingDestinationId");
+    })
+  );
+
+  promiseArray.push(
+    knex.schema.table("Routing_Rules", t => {
+      t.renameColumn("RoutingRuleSetId", "routingRuleSetId");
+      t.renameColumn("RoutingDestinationId", "routingDestinationId");
+    })
+  );
+
+  promiseArray.push(
+    knex.schema.table("Sections", t => {
+      t.renameColumn("QuestionnaireId", "questionnaireId");
+    })
+  );
+
+  promiseArray.push(
+    knex.schema.table("Validation_AnswerRules", t => {
+      t.renameColumn("AnswerId", "answerId");
+    })
+  );
+
+  promiseArray.push(
+    knex.schema.table("Metadata", t => {
+      t.renameColumn("QuestionnaireId", "questionnaireId");
+    })
+  );
+
+  return Promise.all(promiseArray);
+};
+
+exports.down = function(knex) {

```

Comment:
Do we need this?

Reply:
Probably not, pretty much just added it for the sake of consistency, worth having that tech session soon so we can decide whether to be rid of them for good.